src/sentry/seer/anomaly_detection/utils.py

```diff
@@ -75,23 +109,16 @@ def get_crash_free_historical_data(
     )
 
 
-def format_historical_data(data: SnubaTSResult, dataset: Any) -> list[TimeSeriesPoint]:
+def format_historical_data(
+    data: SnubaTSResult, query_columns: list[str], dataset: Any, organization: Organization
+) -> list[TimeSeriesPoint]:
     """
     Format Snuba data into the format the Seer API expects.
-    For errors/transactions data:
-        If there are no results, it's just the timestamp
-        {'time': 1719012000}, {'time': 1719018000}, {'time': 1719024000}
-
-        If there are results, the aggregate is added
-        {'time': 1721300400, 'count': 2}
-
-    For metrics_performance dataset/sessions data:
-        The count is stored separately from the timestamps, if there is no data the count is 0
     """
     formatted_data: list[TimeSeriesPoint] = []
-    nested_data = data.data.get("data", [])
 
     if dataset == metrics_performance:
+        nested_data = data.data.get("data", [])
```

Comment:
so, this is... `data.data.data`? 😅 is there anywhere we have control over any of those that we could rename the variable? i think adding context like what kind of data it is would help a lot.

Reply:
Sadly no, this is the format of `SnubaTSResult` which is typed in the function signature.